docker/files/etc/nginx/conf.d/default.conf

```
@@ -34,6 +41,10 @@ server {
     }
 
     location /collaboration-auth {
+        proxy_cache auth_cache;
+        proxy_cache_key "$http_authorization";
```

Maybe add something more specific to avoid sharing the same cache key later with an other location

```suggestion
        proxy_cache_key "$http_authorization$request_uri";
```

---

Yes you'r totally right